requirements.txt

```diff
@@ -1,4 +1,9 @@
 asgiref==3.5.2
-Django==2.2
+backports.zoneinfo==0.2.1
```

Comment:
```suggestion
backports.zoneinfo==0.2.1;python_version<"3.9"
```

Reply:
Requesting this change to avoid installing `backports.zoneinfo` when using python >= 3.9. See [here](https://stackoverflow.com/questions/71712258/error-could-not-build-wheels-for-backports-zoneinfo-which-is-required-to-insta)